tso-server/bin/omid.sh

```diff
@@ -39,6 +46,12 @@ for j in ../lib/*.jar; do
     CLASSPATH=$CLASSPATH:$j
 done
 
+if [ -z ${OMID_OPTS+x} ]; then
```

Comment:
what if we do something like 
```
OPTS=$OMID_OPTS  
if [ -z ${OMID_OPTS+x} ]; then  
OPTS="$HBASE_OPTS":"$OPTS";  
```

I am just thinking aloud here.

Reply:
The problem is that both may have similar options, like -Xmx, in which case it depends on the JVM version if the first or last one gets used.

That's why I want to make sure that HBASE_OPTS is a fallback to OMID_OPTS rather than using both.

Phoenix currently adds both, but I also want to fix that one, because of the possible conflicting options.